src/target.cpp

```diff
@@ -698,7 +698,11 @@ Error target_parse_triple(ZigTarget *target, const char *triple) {
     }
 
     target->vendor = ZigLLVM_UnknownVendor;
-    target->is_native = false;
+    ZigTarget native;
+    get_native_target(&native);
+    target->is_native =
```

Comment:
Thanks!

This was me 'fixing' `skipping execution because it is non-native`, and not realizing it was already correct behavior. So I've simply deleted this logic